setup.py

```diff
@@ -42,12 +44,15 @@
     'python-neutronclient>=6.0.0',
     'python-keystoneclient>=3.13.0'
 ]
-REQS_FULL = REQS_BASE + REQS_AWS + REQS_AZURE + REQS_OPENSTACK
+REQS_FULL = REQS_BASE + REQS_AWS + REQS_AZURE + REQS_GCP + REQS_OPENSTACK
 # httpretty is required with/for moto 1.0.0 or AWS tests fail
 REQS_DEV = ([
     'tox>=2.1.1',
     'nose',
     # 'moto>=1.1.11',  # until https://github.com/spulec/moto/issues/1396
+    'docutils>=0.14',
+    'imagesize>=0.7.1',
+    'jinja2>=2.9.6',
```

Comment:
@baizhang @chiniforooshan @mbookman Also these - perhaps a comment indicating what they are used for?